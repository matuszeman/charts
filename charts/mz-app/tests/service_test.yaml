suite: service
values:
  - ../values-ci.yaml
templates:
  - service.yaml
tests:
  - it: should match snapshot with values-ci.yaml
    asserts:
      - matchSnapshot: {}
