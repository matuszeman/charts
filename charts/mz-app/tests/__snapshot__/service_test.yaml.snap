should match snapshot with values-ci.yaml:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: app-name
        app.kubernetes.io/version: 0.1.0
        helm.sh/chart: mz-app-0.1.0
      name: RELEASE-NAME-resource-suffix
    spec:
      ports:
        - name: app
          port: 80
          protocol: TCP
          targetPort: apache
        - name: anotherport
          port: 9000
          targetPort: anotherport
      selector:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: app-name
      type: ClusterIP
